src/be_txn.erl

```diff
@@ -43,7 +43,11 @@ to_type(blockchain_txn_dc_coinbase_v1) ->
 to_type(blockchain_txn_token_burn_exchange_rate_v1) ->
     "token_burn_exchange_rate_v1";
```

Comment:
Is this still valid?

Reply:
Well not quite but we have it in the core anyway, hence I put it here. We can get back to it when DCs gain value.